src/types/modules/electron.d.ts

```diff
@@ -3,7 +3,7 @@ type ContextBridgeApi = {
     send: (channel: string, data?: unknown) => void;
     sendSync: (channel: string, data?: unknown) => unknown;
     invoke: (channel: string, ...args: unknown) => Promise<unknown>;
-    on: (channel: string, func: () => void) => void;
+    on: (channel: string, func: (args) => void) => void;
```

Comment:
`args` should be typed like in the functions above.

Reply:
@allroundexperts did you mean this way?
```
    on: (channel: string, func: (...args: unknown) => void) => void;
```

Do you know how to use them?

I've tried `window.electron.on(ELECTRON_EVENTS.DOWNLOAD_COMPLETED, ({url}) => {`
```
Argument of type '({ url }: { url: any; }) => void' is not assignable to parameter of type '(...args: unknown) => void'.
  Types of parameters '__0' and 'args' are incompatible.
    Type 'unknown' is not assignable to type '[{ url: any; }]'.
```

Reply:
We can use this way, but will need eslint-disable comment. is that ok?

```
    on: (channel: string, func: (...args: unknown[]) => void) => void;
```

```
        // eslint-disable-next-line @typescript-eslint/no-explicit-any
        window.electron.on(ELECTRON_EVENTS.DOWNLOAD_COMPLETED, (args: any) => {
            if (args.url !== url) {
                return;
            }
            resolve();
        });

```

Reply:
Issue resolved, fix found.